.changesets/do-not-report-error-causes.md

```diff
@@ -0,0 +1,24 @@
+---
+bump: minor
+type: change
+---
+
+Do not report error causes if the wrapper error has already been reported. This deduplicates errors and prevents the error wrapper and error cause to be reported separately, as long as the error wrapper is reported first.
```

Comment:
Aside from the specific issue we're encountering with ActionView, for the general case, isn't it more likely that the wrapper is reported _after_ the cause, not before? I'm thinking about scenarios where lower-level errors are wrapped in higher levels to add context.

I think it would be preferable if we had a solution to this problem that wasn't dependent on the order in which they are reported.

Reply:
That's not the behavior I'm seeing here. It reports the Action View wrapper error, not the cause.

Not sure how you want it to be not order dependent. When the error is reported we can't know if it will be an error cause on error wrapper yet.

Reply:
Yes, I agree that it's likely not worth the added complexity. I suppose we could remove errors from the error set when we see an error that wraps them.